toolbox/util/githubClient.go

```diff
@@ -96,9 +96,76 @@ func (g GithubClient) CreatePullRequest(
 	pr, _, err := g.client.PullRequests.Create(
 		context.Background(), g.owner, repo, &req)
 	if err != nil {
-		return err
+		return nil, err
 	}
 	log.Printf("Created new PR at %s", *pr.HTMLURL)
+	return pr, nil
+}
+
+// MergePullRequestOnceCIGreen polls CI status of a PR periodically
```

Comment:
You don't need to pull pr status and wait for ci green by yourself. What you need to do is add those two labels when you create that kind of pr. Mungegithub will be there checking ci status.

Reply:
Yes don't do the merge yourself.

Reply:
You are right! The istio-merge-bot does the check for me. Now I feel stupid.

Reply:
You are right! istio-merge-bot will check the status for me. Now I feel stupid.